.github/scripts/downstream-project-spl-common.sh

```diff
@@ -25,5 +25,11 @@ fi
 
 ./patch.crates-io.sh "$SOLANA_DIR"
 
+echo "[profile.dev.package.curve25519-dalek]
+opt-level = 3" >> Cargo.toml
```

Comment:
Rather than overriding this, let's wait for https://github.com/solana-labs/solana-program-library/pull/7220 to land

Reply:
sure! looks like it has been merged 🫡